api/app/Generators/PoolCandidateCsvGenerator.php

```diff
@@ -521,15 +521,15 @@ private function buildQuery()
         ]);
 
         $this->applyFilters($query, [
-            'priorityWeight' => 'candidateCategory',
-            'poolCandidateStatus' => 'poolCandidateStatuses',
-            'pools' => 'availableInPools',
+            'priorityWeight' => 'whereCandidateCategoryIn',
+            'poolCandidateStatus' => 'whereStatusIn',
+            'pools' => 'whereAvailableInPools',
             'skills' => 'skillsAdditive',
             'community' => 'candidatesInCommunity',
```

Comment:
These should be updated too right?
`whereSkillsAdditive`
`whereHasPoolCandidateCommunity`

Reply:
Fixed as part of [add missing scopes to map](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13511/commits/660f9e37fedd5e714de8b9688430189b40e4b9da)